### ISSUE TITLE
Enable use w/o modifier key

### ISSUE THREAD

**Issue by mwarkentin** [mgtrrz/amazonwebsearch#1]
Some people like to alias `/` as a "search" key, but I don't think there's any way to do this without a modifier key at this point?

**Comment by mgtrrz** [comment]
Correct, the modifier is required but it's definitely doable to assign a single key to it. I'd just need to make sure to add checks so that we're not unintentionally triggering the Services menu when we're typing in a text box
